packages/sdk/src/packageMetadata.ts

```diff
@@ -2,4 +2,4 @@
 // Licensed under the MIT license.
 
 export const name = "@microsoft/teamsfx";
-export const version = "0.1.2";
+export const version = "0.1.3";
```

Comment:
What is the relationship between this version and the package publish?

Reply:
Hi @zhenjiao-ms, this version should be aligned with SDK version in `packages/sdk/package.json`. So this change is to prevent unit test failure in CI because SDK version has already been updated by `lerna publish`. We still have difficulties reading version directly from package.json and manually maintain the consistency in PR. We'll automate this step soon.